test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -80,6 +86,133 @@ public void TextFeaturizerWithWordFeatureExtractorTest()
             Assert.Equal(expected, prediction.Features);
         }
 
+        [Fact]
+        public void TextFeaturizerWithWordFeatureExtractorTestWithNullInputNames()
+        {
+            var data = new[] { new TestClass2() { Features = "This is some text in english", OutputTokens=null},
+                               new TestClass2() { Features = "This is another example", OutputTokens=null } };
+            var dataView = ML.Data.LoadFromEnumerable(data);
+
+            var options = new TextFeaturizingEstimator.Options()
+            {
+                WordFeatureExtractor = new WordBagEstimator.Options() { NgramLength = 1 },
+                CharFeatureExtractor = null,
+                Norm = TextFeaturizingEstimator.NormFunction.None,
+                OutputTokensColumnName = "OutputTokens"
+            };
+
+            var pipeline = ML.Transforms.Text.FeaturizeText("Features", options, null);
```

Comment:
>, null [](start = 79, length = 6)

Since it is a `params` argument, you can omit the argument altogether: `ML.Transforms.Text.FeaturizeText("Features", options)'`.

Reply:
Thanks for the feedback. I already provided another test to test when no inputColumnNames is provided (TextFeaturizerWithWordFeatureExtractorTest**WithNoInputNames**). Here in "TextFeaturizerWithWordFeatureExtractorTest**WithNullInputNames**" I provide the case when a null reference is passed as InputColumnNames.

I do this since the [documentation ](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.textcatalog.featurizetext?view=ml-dotnet
) explicitly states that "[when the inputColumnNames is] set to null, the value of the outputColumnName will be used as source.", whereas the issue #3992 refers to the case where no inputColumnName is provided.

Reply:
So, should I leave my pull request as it is regarding this comment, or should I also remove the code and the test case that cover when an user passes a Null as columnInputNames? @yaeldekel

Reply:
Hi Antonio, thanks for the explanation. Technically, this test (with the explicit null) checks the `inputColumnNames == null` code path, and the other one (with nothing provided) checks the `inputColumnNames.Length == 0` code path, so you can leave them both if you prefer, I don't have a strong preference.

---
In reply to: [324824556](https://github.com/dotnet/machinelearning/pull/4211#discussion_r324824556) [](ancestors = 324824556)

Reply:
Ok, I will leave them both. Thanks for the feedback.